### PR TITLE
180 add reset button

### PR DESCRIPTION
**Pull request by NazarPolevyi**

### CODE REVIEW

File: src/app/specific-group/specific-plan/specific-plan.component.ts [hunk 1]
```diff
@@ -30,21 +30,29 @@ export class UsersWithTasks {
   usertasks: UserTask[];
 }
 
+enum States{
```
Comment (YuLad) [inline src/app/specific-group/specific-plan/specific-plan.component.ts]: move this `enum` into a separate file, export it from file and import here

File: src/app/specific-group/specific-plan/specific-plan.component.ts [hunk 2]
```diff
@@ -30,21 +30,29 @@ export class UsersWithTasks {
...
+  done = 'D',
+  inProgress = 'P',
+  approved = 'A',
+  rejected = 'R',
+  reseted = 'RE',
```
Comment (YuLad) [inline src/app/specific-group/specific-plan/specific-plan.component.ts]: `reseted` -> simply `reset`

File: src/app/specific-group/specific-plan/specific-plan.component.ts [hunk 3]
```diff
@@ -30,21 +30,29 @@ export class UsersWithTasks {
...
-  inProgress = 'P';
-  approved = 'A';
-  rejected = 'R';
+ 
+ 
   panelOpenState = false;
   is_student = true;
   sections: Section[];
-  states: any;
+  states = States;
```
Comment (YuLad) [inline src/app/specific-group/specific-plan/specific-plan.component.ts]: no need to do this - remove

File: src/app/specific-group/specific-plan/specific-plan.component.ts [hunk 4]
```diff
@@ -76,12 +84,12 @@ export class SpecificPlanComponent implements OnInit {
...
     if (event.checked) {
-      this.sections[sectionId].Content.UsersTasks[taskId].UserTasks[0].State = this.done;
-      this.taskService.updateUserTaskState(event.source.id, this.done).subscribe();
+      this.sections[sectionId].Content.UsersTasks[taskId].UserTasks[0].State = this.states.done;
+      this.taskService.updateUserTaskState(event.source.id, this.states.done).subscribe();
       this.setPictureState(sectionId, taskId);
     } else {
-      this.sections[sectionId].Content.UsersTasks[taskId].UserTasks[0].State = this.inProgress;
-      this.taskService.updateUserTaskState(event.source.id, this.inProgress).subscribe();
+      this.sections[sectionId].Content.UsersTasks[taskId].UserTasks[0].State = this.states.inProgress;
```
Comment (YuLad) [inline src/app/specific-group/specific-plan/specific-plan.component.ts]: simply use `States.inProgress`

Comment (NazarPolevyi) [reply src/app/specific-group/specific-plan/specific-plan.component.ts]: Ok, I fix it

**Pull request merged**